jwst/resample/resample_spec.py

```diff
@@ -183,23 +298,14 @@ def build_interpolated_output_wcs(self, refmodel=None):
         # compute the output array size in WCS axes order, i.e. (x, y)
         output_array_size = [0, 0]
         output_array_size[spectral_axis] = len(wavelength_array)
-        output_array_size[spatial_axis] = len(x_tan_array)
-
+        output_array_size[spatial_axis] = x_size
         # turn the size into a numpy shape in (y, x) order
         self.data_size = tuple(output_array_size[::-1])
         bounding_box = resample_utils.wcs_bbox_from_shape(self.data_size)
         output_wcs.bounding_box = bounding_box
 
         return output_wcs
 
-    def blend_output_metadata(self, output_model):
```

Comment:
Why is the blending operation being removed? We still need that? I noticed that the import of blendmeta was removed up above too. Is this just a conflict in code versions?